src/apis/print.js

```diff
@@ -16,6 +16,8 @@ WebViewer(...)
 import print from 'helpers/print';
 import selectors from 'selectors';
 
-export default store => () => {
-  print(store.dispatch, selectors.isEmbedPrintSupported(store.getState()));
+export default store => {
+  window.docViewer.getAnnotationManager().getFieldManager().setPrintHandler(() => {
```

Comment:
If we change this file won't this change it so that anyone who calls `instance.print()` won't actually set the print modal, it will just set this function?

Reply:
I didn't see the old function `instance.print()` ever being used, so I thought we can just replace it. Let's chat in person when @mparizeau-pdftron you have time.

Reply:
After discussing with Matt, functions under APIs could possible be called by our clients, so I am reverting this function and move `setPrintHandler()` to `src/index.js`